build/Build.cmd

```diff
@@ -81,13 +81,15 @@ IF "%APPVEYOR_REPO_TAG%" == "true" (goto :sign)
 copy /y %temp%\pom.xml.original pom.xml
 
 if %ERRORLEVEL% NEQ 0 (
-    @echo Build SparkCLR Scala components failed, stop building.
-    popd
-    goto :eof
+  @echo Build SparkCLR Scala components failed, stop building.
+  popd
+  goto :eof
 )
 
 @echo SparkCLR Scala binaries
 copy /y target\*.jar "%SPARKCLR_HOME%\lib\"
+@REM Exclude any original pre-shaded .jar files
```

Comment:
When i run the build\Build.cmd script, there are two .jar files placed in the runtime\lib directory - the "original" [pre-shaded] and the final [uber / shaded] jar.

I believe we only want the latter in the runtime directory tree?

```
Directory of E:\Depot\GitHub\SparkCLR\build\runtime\lib

01/21/2016  09:50 AM           159,661 original-spark-clr_2.10-1.6.0-SNAPSHOT.jar
01/21/2016  09:50 AM        32,835,116 spark-clr_2.10-1.6.0-SNAPSHOT.jar
```